includes/integration.php

```diff
@@ -799,20 +804,25 @@ public function new_order_event($order_id){
         $order = new WC_Order($order_id);
 
         $call_params = false;
-
-        if(!$this->stringIsPresent(get_post_meta($order_id, '_billing_email', true))) {
+    
+        if(!$this->stringIsPresent(get_post_meta($order_id, '_billing_email', true)) && !$this->stringIsPresent(get_post_meta($order_id, '_billing_phone', true))) {
```

Comment:
Same here. Get those values in `$email` and `$phone` variables and use them after that also (lines 811 on).

Reply:
Refactored.